CHANGELOG.txt

```diff
@@ -2,6 +2,20 @@
 TextWiser CHANGELOG
 =====================
 
+-------------------------------------------------------------------------------
+Mar 03, 2022 1.4.0
+-------------------------------------------------------------------------------
+major:
```

Comment:
Not a must, just mentioning as a nice-to-have, opportunity, IF easy, linking to code for some (major) updates might be good. Again, not necessary, just mentioning it. For instance, update umap (what file/method is that even if it is not exact line)